### PR TITLE
Contact delete integration

### PR DESCRIPTION
**Pull request by gsplakmal**
Contact delete
Error handling updates

### CODE REVIEW

File: src/features/contacts/index.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export const Contacts = () => {
             <Avatar src={row.avatarUrl!}></Avatar>
           </ListItemAvatar>
           <ContactNameListItemText
-            primary={`${row.firstName} ${row.lastName}`}
+            primary={`${row.firstName ? row.firstName : ""} ${row.lastName ? row.lastName : ""}`}
```
Comment (idchrono) [inline src/features/contacts/index.tsx:54]: you can make it easier like
row.firstName || ""

File: src/features/contacts/view/index.tsx [hunk 2]
```diff
@@ -14,7 +14,9 @@ import { BreadCrumbNavigation } from "components/breadcrumbs";
 export const ContactBase = () => {
   const { state } = useLocation();
   const contact = state as ContactDetailsDto;
-  const contactFullName = `${contact.firstName} ${contact.lastName}`;
+  const contactFullName = `${contact.firstName ? contact.firstName : ""} ${
```
Comment (idchrono) [inline src/features/contacts/view/index.tsx:17]: as before you can make it easier like
contact.firstName || "

**Review by idchrono — COMMENTED** [review]
please see comments

File: src/features/contacts/index.tsx (hunk 1 above)
Comment (gsplakmal) [reply src/features/contacts/index.tsx]: Fixed it.

File: src/features/contacts/view/index.tsx (hunk 2 above)
Comment (gsplakmal) [reply src/features/contacts/view/index.tsx]: Fixed it.

**Pull request merged**